src/itinerary/index.tsx

```diff
@@ -1,150 +1,196 @@
-            ...link.props,
-            className: cc(['kirk-itinerary-location-wrapper', link.props.className]),
-          }
-        } else if (typeof link === 'string') {
-          Component = 'a'
-          chevron = true
-          hrefProps = {
-            href: place.href,
-            className: 'kirk-itinerary-location-wrapper',
-          }
-        } else {
-          Component = 'div'
-          hrefProps = {
-            className: 'kirk-itinerary-location-wrapper',
-          }
-        }
-
-        return (
-          <li
-            className={cc([
-              'kirk-itinerary-location',
-              {
-                'kirk-itinerary--departure': index === 0,
-                'kirk-itinerary--arrival': isLastPlace,
-                'kirk-itinerary-location--toAddon': isLastPlace && isNonEmptyString(toAddon),
-              },
-            ])}
-            key={key}
-            itemProp="location"
-            itemScope
-            itemType="http://schema.org/Place"
-          >
-            <meta itemProp="name" content={place.mainLabel} />
-            <meta
-              itemProp="address"
-              content={
-                place.subLabel && typeof place.subLabel === 'string'
-                  ? place.subLabel
-                  : place.mainLabel
-              }
-            />
-            <Component {...hrefProps}>
-              {!isSmall && (
-                <time dateTime={place.isoDate}>
-                  <Text display={TextDisplayType.TITLESTRONG}>{place.time}</Text>
-                </time>
-              )}
-              <div className="kirk-itinerary-location-city">
-                <Text display={TextDisplayType.TITLESTRONG}>{place.mainLabel}</Text>
-                {!isSmall &&
-                  place.subLabel &&
-                  (typeof place.subLabel === 'string' ? (
-                    <Text
-                      tag={TextTagType.PARAGRAPH}
-                      display={TextDisplayType.CAPTION}
-                      textColor={color.primaryText}
-                    >
-                      {place.subLabel}
-                    </Text>
-                  ) : (
-                    <div>{place.subLabel}</div>
-                  ))}
-              </div>
-              {chevron && (
-                <div className="kirk-itinerary-location-chevron">
-                  <ChevronIcon />
-                </div>
-              )}
-            </Component>
-          </li>
-        )
-      })}
-      {isNonEmptyString(toAddon) && (
-        <li className="kirk-itinerary-toAddon">
-          <Text display={TextDisplayType.CAPTION}>{toAddon}</Text>
-        </li>
-      )}
-      <style jsx>{style}</style>
-    </ul>
-  )
-}
-
-export default Itinerary
+import styled from 'styled-components'
+import { color, space, componentSizes } from '_utils/branding'
+
+import Itinerary from './Itinerary'
+
+const distanceFromHeight = '40px'
+
+const StyledItinerary = styled(Itinerary)`
+  li {
```

Comment:
no `&` ?

Reply:
`&` is on the `<ul>` but the base class is never used directly, only with modifiers (`--small`)